index.html

```diff
@@ -190,37 +223,78 @@ <h4 class="price__title">Wade Warren</h4>
         </section>
       </div>
       <div class="section footer">
-        <div class="cards contact">
-          <p class="contact__title">CONTACTO</p>
-          <h2 class="title footer__title">
-            ¿Deseas realizas un pedido especial?
-          </h2>
-          <div class="boxInformation">
-            <img src="./icons/Icon-phone.svg" alt="" />
-            <p class="textInformation">+1 (234) 928-22-34</p>
+        <div class="footer__box">
+          <div class="cards contact">
+            <p class="contact__title">CONTACTANOS</p>
+            <h2 class="title footer__title">
+              ¿Deseas realizas un pedido especial?
+            </h2>
+            <div class="boxInformation">
+              <img class="icon" src="./icons/Icon-phone.svg" alt="" />
+              <p class="textInformation">+1 (234) 928-22-34</p>
+            </div>
+            <div class="boxInformation">
+              <img class="icon" src="./icons/Icon-email.svg" alt="" />
+              <p class="textInformation">cakeshop@gmail.com</p>
+            </div>
           </div>
-          <div class="boxInformation">
-            <img src="./icons/Icon-email.svg" alt="" />
-            <p class="textInformation">cakeshop@gmail.com</p>
+          <div class="fill">
+            <div class="fill__content">
+              <div>
+                <h4 class="price__title">
+                  Envía un mensaje con los detalles de tu pedido
+                </h4>
+              </div>
+              <div class="put name">
+                <label for="name" class="fill__title">NOMBRE COMPLETO</label>
+                <input
+                  class="holder name"
+                  type="text"
+                  name="name"
+                  id="name"
+                  placeholder="Ingresa tu nombre"
+                />
+                <span class="error" id="nameError"></span>
+              </div>
+              <div class="put email">
+                <label for="email" class="fill__title"
+                  >CORREO ELECTRONICO</label
+                >
+                <input
+                  class="holder email"
+                  type="text"
+                  name="email"
+                  id="email"
+                  placeholder="Ingresa tu correo electrónico"
+                />
+                <span class="error" id="emailError"></span>
+              </div>
+              <div class="put descripcion">
+                <label for="descripción" class="fill__title">DESCRIPCION</label>
+                <input
+                  class="holder holder__descripcion"
+                  type="text"
+                  name="descripcion"
+                  id="descripcion"
+                  placeholder="Ingresa una breve descripción del pedido que tienes en mente"
+                />
+                <span class="error" id="descripcionError"></span>
+              </div>
+              <button class="button enviar">ENVIAR MENSAJE</button>
```

Comment:
add form validation:
- Name should be more than 4 characters
- Email should contain `@`
- Description should be more than 10 characters